quetz/testing/fixtures.py

```diff
@@ -147,8 +148,35 @@ def expires_on_commit():
 
 
 @pytest.fixture
-def db(session_maker, expires_on_commit):
+def db(session_maker, expires_on_commit, auto_rollback, request):
```

Comment:
Thinking out loud: Instead of changing this fixture in `quetz/testing/fixtures`, would it make sense to redefine it in the `conftest.py` of our plugin?

Reply:
I don't think it matters. I can document this a bit more, and then it might even be useful for other plugins.

Reply:
Motivation: What if we wanted to move the plugin into a separate repository? In that case, we can't just edit fixtures defined inside `quetz`. Our only possibility would be to redefine the fixture, right?

It would be nice to have the plugin in a state where we can quickly move it into a separate repository, which I why I suggested this.

Reply:
That makes sense 👌🏼